### ISSUE TITLE
Consider adding GTSE to PullRequest builder

### ISSUE THREAD

**Issue by sprocter** [osate/osate2#1606]
In order to detect and block pulls that break the GTSE work -- or rather, that break the APIs that GTSE relies upon -- we should consider adding GTSE to the build that gets executed when a new pull request is submitted. If it adds a significant amount of time to the build, however, it may not be worth it.

Note that this depends on #1605.

**Comment by lwrage** [comment]
This would add another repo to the build, which complicates the build. I think it's not worth the effort.
